### DISCUSSION TITLE
Icon request: `layer` (with two layers and no symbols)

### DISCUSSION THREAD

**Discussion by nick-w-nick** [FortAwesome/Font-Awesome#21108] [Icon Requests]
### Use case

To indicate layered/nested or collapsed content. Font Awesome already has pretty much this exact icon, but all of them have symbols (plus, minus, etc.) next to them. There is the `layer-group` icon, which doesn't have any symbols next to it, but it is three layers instead of two.

![image](https://github.com/FortAwesome/Font-Awesome/assets/43578531/f6517927-5c84-4975-b695-8b8e369517b5)

### Example image

![image](https://github.com/FortAwesome/Font-Awesome/assets/43578531/02a92f7f-3abf-49ef-b250-5f1f3fd76fa3)

### Icon request checklist

- [X] The title starts with "Icon request: " and is followed by the requested icon name
- [X] This icon does not represent a brand
- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The request is for a concrete object, or I've included an example image
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/new#faqs)

**Comment by sensibleworld** [comment]
This was just added: https://fontawesome.com/icons/layer?f=classic&s=solid
